playground/infrastructure/helper.py

```diff
@@ -273,33 +276,38 @@ def get_supported_categories(categories_path: str) -> List[str]:
         yaml_object = yaml.load(supported_categories.read(), Loader=yaml.SafeLoader)
         return yaml_object[TagFields.categories]
 
+def _get_url_vcs(filepath: str):
+    """
+    Construct VCS URL from example's filepath
+    """
+    root_dir = os.getenv("BEAM_ROOT_DIR", "")
+    file_path_without_root = filepath.replace(root_dir, "", 1)
+    if file_path_without_root.startswith("/"):
+        return "{}{}".format(Config.URL_VCS_PREFIX, file_path_without_root)
+    else:
+        return "{}/{}".format(Config.URL_VCS_PREFIX, file_path_without_root)
```

Comment:
I know this is a lift and shift, but should we just use `filepath.Join`? https://pkg.go.dev/path#Join

I think the if else could be condensed to `return filepath.Join(Config.URL_VCS_PREFIX, file_path_without_root)`

Reply:
The result here is not a file system path, but URL, though I brushed it up a bit

Reply:
Oh good point.